tests/load/test_s3_modin.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Optional
+
+import modin.pandas as pd
+import pytest
+import ray
+
+import awswrangler as wr
+
+from .._utils import ExecutionTimer
+
+
+@pytest.fixture(scope="function")
+def df_s() -> pd.DataFrame:
+    # Data frame with 100000 rows
+    ray_ds = ray.data.read_parquet("s3://ursa-labs-taxi-data/2010/02/data.parquet")
+    return ray_ds.to_modin()
+
+
+@pytest.fixture(scope="function")
+def big_modin_df() -> pd.DataFrame:
+    pandas_refs = ray.data.range_table(100_000).to_pandas_refs()
+    dataset = ray.data.from_pandas_refs(pandas_refs)
+
+    frame = dataset.to_modin()
+    frame["foo"] = frame.value * 2
+    frame["bar"] = frame.value % 2
+
+    return frame
```

Comment:
Since these are defined in `test_s3` as well, should we just move them into a fixture file?